### PR TITLE
KEYCLOAK-11155 split on first '=' instead of all

### PR DESCRIPTION
**Pull request by edewit**

### CODE REVIEW

File: adapters/oidc/js/src/main/resources/keycloak.js [hunk 1]
```diff
@@ -18,9 +18,9 @@
 (function(root, factory) {
```
Comment (jonkoops) [inline adapters/oidc/js/src/main/resources/keycloak.js]: @edewit Could you remove the unrelated whitespace changes from your commit?

Comment (edewit) [reply adapters/oidc/js/src/main/resources/keycloak.js]: you want to have spaces on the end of lines?

Comment (jonkoops) [reply adapters/oidc/js/src/main/resources/keycloak.js]: No, just that the commit only includes the lines that are actually changed.

Comment (edewit) [reply adapters/oidc/js/src/main/resources/keycloak.js]: just add `?w=1` on the url and you'll see only the important changes

**Comment by edewit** [comment]
@jonkoops removed whitespace changes and revert my editor setting that removes trailing whitespace on save

File: testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/javascript/JavascriptAdapterTest.java [hunk 2]
```diff
@@ -474,6 +474,23 @@ public void testLocationHeaderInResponse() {
                         });
     }
 
+    @Test
+    public void equalsSignInRedirectUrl() {
+
```
Comment (mhajas) [inline testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/javascript/JavascriptAdapterTest.java]: Remove empty line

**Review by mhajas — CHANGES_REQUESTED** [review]
Thank you @edewit for the PR. The fix looks good to me, however I added some comments to test.

File: testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/javascript/JavascriptAdapterTest.java [hunk 3]
```diff
@@ -474,6 +474,23 @@ public void testLocationHeaderInResponse() {
...
+        testAppUrl = authServerContextRootPage.toString().replace("localhost", NIP_IO_URL) + JAVASCRIPT_URL + "/index.html?test=bla=bla&super=man";
```
[the remark this answers is not in the document]
Comment (edewit) [reply testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/javascript/JavascriptAdapterTest.java]: It's done [here](https://github.com/keycloak/keycloak/blob/303514c73fd5b8fafcb9602c73f4b6fdafc6774a/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/javascript/JavascriptAdapterTest.java#L508) as well. How do I set the `testAppUrl` by using `AuthServerTestEnricher.getAuthServerContextRoot` method?

[the remark this answers is not in the document]
Comment (mhajas) [reply testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/javascript/JavascriptAdapterTest.java]: I see. It is working because testAppUrl is assigned in `@Before` method, so it is same for each test. However it is useless to reassign it before each method since for majority of methods it is same. We can have something like this
```
private static final String TEST_APP_URL = AuthServerTestEnricher.getAuthServerContextRoot().replace("localhost", NIP_IO_URL) + JAVASCRIPT_URL + "/index.html";
```
instead of
```
private String testAppUrl;
```
and then create a new variable based on TEST_APP_URL everywhere we need to change it. 

However this is little bit out of this issue and the test will be working even without it, so I won't insist on changing it :-). Feel free to ignore this comment.

[the remark this answers is not in the document]
Comment (edewit) [reply testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/javascript/JavascriptAdapterTest.java]: Right, I feel like this is a separate issue

**Review by mhajas — APPROVED** [review]
@edewit Thank you very much for incorporating comments! I would be happier if you fix also the other thing we were talking about, but I am approving in this state as well.

**Review by jonkoops — APPROVED** [review]
I've tested the changes to the JS locally and they are working great!

**Review by stianst — APPROVED** [review]
